### ISSUE TITLE
Add a unicode modifier for Superscript Star

### ISSUE THREAD

**Issue by jlperla** [JuliaLang/julia#26444]
Right now there is no support for (in latex notation) `x^{*}`. It would be great to have that as a unicode modifier, in the same sense as `Superscript Plus`.

**Comment by JeffBezanson** [comment]
I don't believe there is a Unicode character for this, but we could potentially use the combining asterisk U+20F0 with a space.
